frameworks/g-cloud-10/questions/declaration/servicesHaveOrSupport.yml

```diff
@@ -5,7 +5,7 @@ question: >
     - on-demand self-service. Your cloud services must be automatically available without needing interaction between the supplier and the buyer, for example server time or network storage. (Lot 1 - cloud hosting and lot 2 - cloud software.)
     - broad network access. Your cloud services must be easily accessible over the network and across a wide range of platforms, for example mobile phones, tablets, laptops and workstations. (Lot 1 - cloud hosting and lot 2 - cloud software.)
     - measured service. Your resource use must be monitored, controlled and reported to the buyer, for example storage, processing, bandwidth and number of active user accounts. (Lot 1 - cloud hosting and lot 2 - cloud software.)
-    - the delivery of cloud services. Your service must enable the delivery of cloud services. Your service complies with the definition of cloud support in the <a href="/suppliers/frameworks/g-cloud-10" target="_blank" rel="noopener noreferrer">framework agreement (link opens in a new tab)</a> (Lot 3 - cloud support.)
+    - the delivery of cloud services. Your service must enable the delivery of cloud services. Your service complies with the definition of cloud support in the <a href="/suppliers/frameworks/g-cloud-10#guidance" target="_blank" rel="noopener noreferrer">framework agreement (link opens in a new tab)</a> (Lot 3 - cloud support.)
```

Comment:
Should this be `#legal-documents `?

Reply:
It possibly could be, although @constancecerf seemed happy for us to just link to `#guidance` in all instances.

Reply:
I'll update it as you say, anyway.